parts/kubernetesmastercustomscript.sh

```diff
@@ -19,6 +19,16 @@ SERVICE_PRINCIPAL_CLIENT_SECRET="${11}"
 KUBELET_PRIVATE_KEY="${12}"
 TARGET_ENVIRONMENT="${13}"
 NETWORK_POLICY="${14}"
+# Default values for backoff configuration
+CLOUDPROVIDER_BACKOFF=true
+CLOUDPROVIDER_BACKOFF_RETRIES=6
+CLOUDPROVIDER_BACKOFF_EXPONENT=1.5
+CLOUDPROVIDER_BACKOFF_DURATION=5 # in seconds
+CLOUDPROVIDER_BACKOFF_JITTER=1.0
+# Default values for rate limit configuration
+CLOUDPROVIDER_RATELIMIT=true
+CLOUDPROVIDER_RATELIMIT_QPS=3
+CLOUDPROVIDER_RATELIMIT_BUCKET=10
```

Comment:
TODO: convert the above statically assigned `ENV` values into configurable `ENV` values, i.e., suss out how the `${n}` assignments work.